cmd/nlb.go

```diff
@@ -10,3 +16,19 @@ var nlbCmd = &cobra.Command{
 func init() {
 	RootCmd.AddCommand(nlbCmd)
 }
+
+// lookupNLB attempts to look up an NLB resource by name or ID.
+func lookupNLB(ctx context.Context, zone, ref string) (*egoscale.NetworkLoadBalancer, error) {
```

Comment:
Is it possible to get directly in the API by name?

Reply:
No, ID only.